test-packages/ts-ember-preview-types/app/components/ember-component.hbs

```diff
@@ -0,0 +1,21 @@
+<div ...attributes id={{unique-id}}>
+  {{this.required}}
+  {{this.optional}}
+  {{this.hasDefault}}
+
+  {{@required}}
+  {{@optional}}
+  {{@hasDefault}}
+
+  {{get this "required"}}
```

Comment:
I think this the correct way to test this? /cc @chriskrycho

Reply:
I believe so! And: I will try to help push this the rest of the way across the line today.